setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    version='0.1.0',
+    name='TODO_PROJECT_NAME',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[],
+    description='TODO',
+    author='Incuna Ltd',
+    author_email='admin@incuna.com',
+    url='https://github.com/incuna/TODO_PROJECT_NAME',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```

Comment:
I would probably have a different default for this - a completely new library usually won't start stable. It's a bit different for something pulled out of a project though, so YMMV.